tester_app/client/app/bundles/HelloWorld/startup/helloWorldAppClient.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import createStore from '../store/helloWorldStore';
+import HelloWorld from '../containers/HelloWorld';
+
+const helloWorldAppClient = props => {
```

Comment:
@justin808 I'm confused about this. Being that we aren't passing `generator_function: true` from the view, why does this code work?